routes/routes.go

```diff
@@ -0,0 +1,12 @@
+package routes
+
+import (
+	"github.com/Diegoplas/go-bootcamp-deliverable/controllers"
+	"github.com/gorilla/mux"
+)
+
+func GetRouter() (router *mux.Router) {
+	router = mux.NewRouter()
+	router.HandleFunc("/pokedex/{pokemon_id}", controllers.GetPokemon).Methods("GET")
```

Comment:
Hmm I don't think it's necessary to change the route... but we would like to be as explicit as possible about the resource we're trying to work with, probably a path like `/pokemon/{id}` would be better. This route as you have it makes me think that we're fetching pokedexes, not pokemons